bundles/org.openhab.binding.allplay/README.md

```diff
@@ -123,5 +123,5 @@ rule "Play Online Radio stream"
 when
     Item All2OnlineRadio changed to ON
 then
-    All2Stream.sendCommand("http://chromaradio.com:8008/listen.pls")
+    All2Stream.sendCommand("https://chromaradio.com:8008/listen.pls")
```

Comment:
Are you sure about this one, it looks unexpected

Reply:
Nope not sure, while looking at it again.
I can revert that, since it is in a rule example.